app/controllers/api/v1x0/orders_controller.rb

```diff
@@ -14,6 +14,8 @@ def create
       def submit_order
         approval = Catalog::CreateApprovalRequest.new(params.require(:order_id))
         render :json => approval.process.order
+      rescue Catalog::ApprovalError => e
+        render :json => { :errors => e.message }, :status => :unprocessable_entity
```

Comment:
@lindgrenj6 This is misleading, it has to be Internal Server Error (500), since we did a search for a Approval Workflow at multiple levels and didn't find any matches. The user didn't pass anything in so it can't be any of the 400 errors. The error message could indicate that no approvals were found but its a server side error.

Reply:
Updated to use `:internal_server_error` instead of `:unprocessable_entity`